doc/source/data/transforming-data.rst

```diff
@@ -305,12 +305,64 @@ To transform groups, call :meth:`~ray.data.Dataset.groupby` to group rows. Then,
                 .map_groups(normalize_features)
             )
 
-.. _shuffling_rows:
+.. _shuffling_data:
 
-Shuffling rows
+Shuffling data
 ==============
 
-To randomly shuffle all rows, call :meth:`~ray.data.Dataset.random_shuffle`.
+Shuffle the ordering of files
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To randomly shuffle the ordering of input files before reading, call a function like
+:func:`~ray.data.read_images` and specify ``shuffle="files"``. This randomly assigns
+input files to workers for reading.
+
+.. testcode::
+
+    import ray
+
+    ds = ray.data.read_images(
+        "s3://anonymous@ray-example-data/image-datasets/simple",
+        shuffle="files",
+    )
+
+.. tip::
+
+    This is the fastest option for shuffle, and is purely metadata operation. This
+    option does not shuffle the actual rows inside files, so the randomness would be
```

Comment:
https://developers.google.com/style/contractions#:~:text=In%20particular%2C%20it's%20fine%20to,misread%20don't%20as%20do.
```suggestion
    option doesn't shuffle the actual rows inside files, so the randomness might be
```

Reply:
updated.